server/src/dynamodb.ts

```diff
@@ -23,25 +32,25 @@ export class Dynamo implements Repository {
         },
       };
 
-      const dynamoClient = new AWS.DynamoDB(dynamoConfigOptions);
-      console.debug('dynamo config: ', dynamoConfigOptions);
+      const dynamoClient = new AWS.DynamoDB(this.dynamoConfigOptions);
+      console.debug('dynamo config: ', this.dynamoConfigOptions);
       await dynamoClient.createTable(params).promise();
       const data = await dynamoClient.listTables().promise();
       console.log('Created table.', JSON.stringify(data, null, 2));
     }
   }
 
   public async create(project: DatabaseModel): Promise<string> {
-    const timestamp = new Date().getTime();
+    const date = new Date().toString();
```

Comment:
Probs better to use  `new Date().toISOString()` since it should be more portable